exercises/sum-of-multiples/instructions.md

```diff
@@ -0,0 +1,27 @@
+# Instructions
+
+Your task is to write the code that calculates the energy points that get awarded to players when they complete a level.
+
+The points awarded depend on two things:
+
+- the number of the level that the player completed
+- the base value of each magical item collected by the player during that level
+
+The energy points are awarded according to the following rules:
+
+- For each magical item, take the base value and find all the multiples of that value that are less than or equal to the level number.
+- Combine the sets of numbers.
+- Remove any duplicates.
+- Calculate the sum of all the numbers that are left.
```

Comment:
These read like steps, not arbitrary bullet points or rules.

```suggestion
1. For each magical item, take the base value and find all the multiples of that value that are less than or equal to the level number.
2. Combine the sets of numbers.
3. Remove any duplicates.
4. Calculate the sum of all the numbers that are left.
```

Reply:
Thanks for the tweaks, @SleeplessByte